src/pages/professional/professionalPage.ts

```diff
@@ -0,0 +1,9 @@
+export const PRO_TITLE = 'Professional Version';
+
+export const PRO_TEXT = [
+  'We are actively working on a professional version.The available consumer version is not intended or certified for professional use. We would love to keep in touch via email to let you know when we have a professional version.',
+
+  'Not a professional (e.g. providers office, appeal specialist, lawyer, etc.) but still ended up here? Not to worry! You can try the regular appeal generator here.',
```

Comment:
This seems weird? Like we want this one to be linked to something and the other ones not but.